docs/3.0.0-beta.x/guides/deployment.md

```diff
@@ -1622,3 +1622,389 @@ heroku open
 Like with project updates on Heroku, the file system doesn't support local uploading of files as they will be wiped when Heroku "Cycles" the dyno. This type of file system is called [ephemeral](https://devcenter.heroku.com/articles/dynos#ephemeral-filesystem), which means the file system only lasts until the dyno is restarted (with Heroku this happens any time you redeploy or during their regular restart which can happen every few hours or every day).
 
 Due to Heroku's filesystem you will need to use an upload provider such as AWS S3, Cloudinary, or Rackspace. You can view the documentation for installing providers [here](../plugins/upload.md#install-providers) and you can see a list of providers from both Strapi and the community on [npmjs.com](https://www.npmjs.com/search?q=strapi-provider-upload-&page=0&perPage=20).
+
+## Google App Engine
+
+In this guide we are going to:
+
+- create a new Strapi project
+- configure PostgreSQL for the production enviroment
+- deploy the app to Google App Engine
+  - (using Standard Environment, but Flexible would be possible and probably easier)
+- add the [Google Cloud Storage file uploading plugin](https://github.com/Lith/strapi-provider-upload-google-cloud-storage) by [@Lith](https://github.com/Lith)
+
+### New Strapi project
+
+```
+yarn create strapi-app <YOUR_APP_NAME> --quickstart
+```
+
+The setup:
+
+- Creates the initial files, configured to use `sqlite` database
+- Runs `strapi build "--no-optimization"` to build the admin UI
```

Comment:
When we create a project using the command you specified just before, you don't have to run this command because it's automatic using the `--quickstart`
It's the same for the next line.
I think we can remove this part.